compiler/luci/pass/src/FuseAddWithFullyConnectedPass.cpp

```diff
@@ -24,6 +26,49 @@
 
 namespace
 {
+
+#define RETURN_FALSE_UNLESS(cond) \
+  if (not(cond))                  \
+    return false;
+
+bool fc_with_add_pattern_check(const loco::DataType dtype, luci::CircleAdd **add,
```

Comment:
Q) why use pointers of pointer? `add` is not updated here.
It would be better not to use double pointer if not necessary
and also use reference if need update in this method.

Reply:
@seanshpark as for `add` I fully understand the concern. This is changed.
As for double pointers, it was all required by `luci::fill` to get proper handling (i.e. do not loose pointer locally in the function, which resulted in segfaults). Now I propose use of struct to keep track of nodes in the pattern. I see it as proper middle-ground solution, that also organize the required nodes in one place. What do you think?